frontend/src/components/common/form/Check.tsx

```diff
@@ -95,12 +95,12 @@ export const Check: React.FC<CheckProps> = ({
             isInvalid={!!touch && !!error}
             type={type}
             {...rest}
-            value={!!checked}
+            value={checked === true}
```

Comment:
is there much of a difference between these two?

Reply:
Yep, javascript truthy/falsey checks encompass a wide range of values https://developer.mozilla.org/en-US/docs/Glossary/Falsy

explicitly checking for true/false narrows the number of accepted values considerably

Reply:
thanks!